salt/modules/dockerio.py

```diff
@@ -648,8 +675,9 @@ def create_container(image,
         }
         __salt__['mine.send']('docker.get_containers', host=True)
         return callback(status, id_=container, comment=comment, out=out)
-    except Exception:
+    except Exception, e:
```

Comment:
Hm. We're catching exceptions coming from docker.py and possibly from urllib, etc. I'll poke through the code and see what makes sense.